modules/gapi/src/backends/common/serialization.cpp

```diff
@@ -638,6 +694,12 @@ I::OStream& ByteMemoryOutStream::operator<< (char atom) {
     m_storage.push_back(atom);
     return *this;
 }
+I::OStream& ByteMemoryOutStream::operator<< (wchar_t atom) {
+    for (std::size_t i = 0; i < sizeof(wchar_t); ++i) {
```

Comment:
> sizeof(wchar_t)

`wchar_t` has different sizes on different platforms: 16/32-bits on Windows/Linux. This is important for cross-platform serialization.
`long` has similar issue (just in case).

Reply:
Right. Is there a problem here? I believe `sizeof()` works just fine for cross-platform her

Reply:
Fixed, thanks. Also added a simple test to prevent possible future problems